Week2/homework/exercise2.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+for (var i = 0; i <= 20; i++) {
+    if (i % 2 === 0)
```

Comment:
Ah. The logic is right but I don't think using `var` keyword is the best of ideas. Not sure if you are reading any material outside HYF's which use `var` because I remember David didn't use or recommend using `var` keyword during the classroom sessions. It might be that those materials are a bit out of date. Would recommend you to check the timestamp of those materials. 🙂

Reply:
Yash, you're right. I guess the lots of var on my codes are because when I have issues making my logic works, I google a bunch of questions and get some answers that really helps me but are kinda old, so explains the whole var presence. I'm sorry for that, I'll pay more attention to HYF materials to keep the standart.

Reply:
Ah. I didn't mean that you should not look up on the Internet but rely solely on HYF material. It is good that you are looking at different places to solve your problem. I only wanted to ask you to be conscious of the timeline of the articles which you are referring. 🙂

Reply:
Yeah, I got your point. I'll be more conscious about it for sure and thanks for being so patient because you saw lots of exercises with the same error, but funny thing about it is that yesterday I was doing some challenges on Khan Academy to learn a bit more about the new things we are seeing in JS2 and it didn't accept let or const. If we didn't put var in there is like we are doing wrong. Anyways, now I know that I should avoid and I'll keep avoiding even the challenge obligates me to use hahaha.
![Screen Shot 2019-10-09 at 9 54 22 AM](https://user-images.githubusercontent.com/54168072/66462290-0d81d400-ea7b-11e9-84cd-abc3bb7d3148.png)